backend/src/crd/controller/scheduledworkflow/main.go

```diff
@@ -31,8 +31,9 @@ import (
 )
 
 var (
-	masterURL  string
-	kubeconfig string
+	masterURL         string
+	kubeconfig        string
+	informerNamespace string
```

Comment:
Could you please also do it for the viewer CRD so that folks that implement a new CRD have a consistent pattern to follow?

Reply:
@vicaire Viewer CRD uses another approach (SimpleController) to implement. Not sure how to plugin a namespace into it.. Do you happen to know? @neuromage